packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -79,9 +85,9 @@ Choice.propTypes = {
    */
   inversed: PropTypes.bool,
   /**
-   * Displays the checkbox on the right side of the label
+   * Placement of the input relative to the text label (`left` or `right`)
```

Comment:
![image](https://user-images.githubusercontent.com/371943/30556816-255bb402-9c7a-11e7-8c8f-48ca932e6e87.png)

Nit: The documentation will automatically indicate the possible values based on what you set in `PropTypes.oneOf`, so we could get rid of that info from the description. 

Related: Can we set a `defaultProps` value for `inputPlacement`? Right now it's unclear to someone looking at the documentation or source code what the placement will be if it's left blank.